server/routes/api/comments.js

```diff
@@ -42,4 +43,8 @@ comments.patch('/comments/:commentId',
 comments.get('/comments/:commentId',
   verifyToken, validateResourceId, getComment);
 
+// delete a user comment
+comments.delete('/articles/:articleId/comments/:commentId',
```

Comment:
This route's uri can just be '/comments/:commentId'. There's no need to add the '/articles/:articleId' since when we're deleting, we're only working with the 'commentId'

Reply:
@Darthrighteous I see what you mean, but why it better this way?

Reply:
okay, this makes sense. Thanks.